migration_original/ODS1Stage/tables/Show/SOLRGeographicArea/spu_translated_SOLRGeographicArea.sql

```diff
@@ -0,0 +1,128 @@
+-- spuSOLRGeographicAreaGenerateFromMid
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRGEOGRAPHICAREA() 
```

Comment:
Add the Database to the procesure name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRGEOGRAPHICAREA() 
```

Reply:
Done